Makefile

```diff
@@ -440,7 +451,7 @@ endif
 # Define build targets
 ##############################
 .PHONY: all lib test clean docs linecount lint lintclean tools examples $(DIST_ALIASES) \
-	py mat py$(PROJECT) mat$(PROJECT) proto runtest \
+	py mat py$(PROJECT) mat$(PROJECT) thirdparty proto runtest \
```

Comment:
I'm confused as to why this is here, as `thirdparty` doesn't seem to be referenced later on. Have I missed something?